### PR TITLE
Start using github actions

### PR DESCRIPTION
**Pull request by elfosardo**
We start using github actions for testing
See #165 for more info

### CODE REVIEW

File: .github/workflows/tox-testing.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Tox
+      run: pip install tox
+    - name: Run pep8 tests
+      run: tox -vv -e pep8
+    - name: Run unit tests
+      run: tox -vv -e py
```
Comment (dtantsur) [inline .github/workflows/tox-testing.yml:24]: You don't need to run unit tests if you run coverage tests

File: .github/workflows/tox-testing.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    - name: Run coverage tests
+      run: tox -vv -e cover
+    - name: Run docs tests
+      run: tox -vv -e docs
```
Comment (dtantsur) [inline .github/workflows/tox-testing.yml:28]: Ideally we should run docs and pep8 separately from unit tests.

**Review by dtantsur — CHANGES_REQUESTED** [review]

**Comment by dtantsur** [comment]
Oh, and let's remove travis.yaml?

File: .github/workflows/tox-testing.yml (hunk 2 above)
Comment (elfosardo) [reply .github/workflows/tox-testing.yml:28]: true, I'm going to do a separate job for each

File: .github/workflows/tox-testing.yml (hunk 1 above)
Comment (elfosardo) [reply .github/workflows/tox-testing.yml:24]: right, I'll remove unit tests run from here

**Comment by elfosardo** [comment]
I removed the travis jobs from required and added the github actions

**Comment by elfosardo** [comment]
once this merged I will also remove the webhook to travis-ci

**Comment by elfosardo** [comment]
@ErwanAliasr1 @dtantsur this should be ready to go, please double-check

**Review by iurygregory — APPROVED** [review]

**Pull request merged**